src/receive.js

```diff
@@ -1,12 +1,17 @@
 import Dat from './lib/dat'
 import DatCp from './lib/dat-cp'
 
-export default async function receive(key, options) {
+export default async function receive(args, options) {
+
+  let key = args[0]
+  let path = args[1] ? args[1] : '.'
+
   const dat = await Dat({key, sparse: true})
 
   if (!options.skipPrompt) {
     const datCpDryRun = new DatCp(dat, {...options, dryRun: true})
 
+    await datCpDryRun.setDownloadDest(path)
```

Comment:
This is causing a couple of issues:

* If you do something like `npm run cli -- <key> out -n`, you end up with a `out/out/file`. It only works as expected if you provide the `--skip-prompt` flag.
* A dry run is still creating the out directory. A dry run should make no changes to the filesystem.

I don't think this call actually does anything anyway - I would just remove it.